tests/core/evaluation/test_markers_config.py

```diff
@@ -104,17 +105,74 @@ def test_load_invalid_path():
         MarkerConfig.load_config_from_path("not a path")
 
 
-def test_load_valid_file(simple_marker_config: Text, simple_marker_config_json):
+def test_load_valid_file(simple_markers_config, simple_marker_config_json):
     """Tests the single config loader"""
-    yaml_as_dict = MarkerConfig.load_config_from_path(simple_marker_config)
+    yaml_as_dict = MarkerConfig.load_config_from_path(simple_markers_config)
     assert yaml_as_dict == simple_marker_config_json
 
 
-def test_load_valid_path(multi_marker_config_folder, multi_marker_config_json):
+def test_load_valid_path(markers_config_folder, multi_marker_config_json):
     """Tests the config folder loading"""
-    yaml_as_dict = MarkerConfig.load_config_from_path(multi_marker_config_folder)
+    yaml_as_dict = MarkerConfig.load_config_from_path(markers_config_folder)
     # check that the two configs contain the same entries.
     for m in yaml_as_dict["markers"]:
         assert m in multi_marker_config_json["markers"]
     for m in multi_marker_config_json["markers"]:
         assert m in yaml_as_dict["markers"]
+
+
+def test_valid_config(simple_marker_config_json):
+    assert MarkerConfig._validate_config(simple_marker_config_json) is True
+
+
+def test_valid_config_operators(markers_config_operators):
+    config = MarkerConfig.load_config_from_path(markers_config_operators)
+    assert MarkerConfig._validate_config(config) is True
+
+
+def test_config_missing_required_top_level_markers_label():
+    sample_json = {"marker": "no_restart"}
+    with pytest.raises(ValidationError):
+        MarkerConfig._validate_config(sample_json)
```

Comment:
I would recommend using the public method `from_yaml` directly in each test, since `_validate_config` should not be accessed outside the `MarkerConfig` class.

Reply:
I changed `_validate_config` to a public method. I think it works conceptually as a public method. What do you think?

Reply:
If you expect this method to be accessed in other places in the codebase, then that's fine.